nautobot/dcim/migrations/0009_increase_length_on_all_serials.py

```diff
@@ -0,0 +1,23 @@
+# Generated by Django 3.1.14 on 2021-12-21 20:29
```

Comment:
No need for these to be two separate migrations files - can you please delete and regenerate a single combined migration?